apps/app/src/app/shared/services/auto-refresh.service.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AutoRefreshService } from './auto-refresh.service';
+
+describe('AutoRefreshService', () => {
+  let service: AutoRefreshService;
+  let router: Router;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [RouterTestingModule],
+      providers: [AutoRefreshService],
+    });
+    router = TestBed.get(Router);
+    service = TestBed.inject(AutoRefreshService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should call "disable" when "ngOnDestroy" is called', () => {
```

Comment:
Au lieu de "should call disable" tu devrais plutôt indiquer "should set intervall to null" car sans connaitre le code, on s'attend à ce que tu testes le fait que ça appelle la fonction disable et non à ce que l'interval soit null

Reply:
yes effectivement !